src/ToricVarieties/LineBundles/attributes.jl

```diff
@@ -21,3 +22,36 @@ function variety(l::ToricLineBundle)
     return l.variety
 end
 export variety
+
+
+@doc Markdown.doc"""
+    toric_divisor(l::ToricLineBundle)
+
+Returns a divisor corresponding to the toric line bundle `l`.
+"""
+function toric_divisor(l::ToricLineBundle)
+    return get_attribute!(l, :toric_divisor) do
+        class = divisor_class(l)
+        map1 = map_from_cartier_divisor_group_to_picard_group(variety(l))
+        map2 = map = map_from_cartier_divisor_group_to_torus_invariant_divisor_group(variety(l))
+        image = map2(preimage(map1, class)).coeff
+        coeffs = vec([Int(x) for x in image])
```

Comment:
So `image` is an `fmpz_mat` row vector here. Is it really guaranteed that all entries will fit into an `Int?

Reply:
Perse not. However, I believe we are currently limited to cases for which this cast works. Namely, the list of coefficients is eventually passed to `Polymake`:

> ptd = Polymake.fulton.TDivisor(COEFFICIENTS=coeffs)

If I call this with `coeffs = [fmpz(1), fmpz(2)]`, then this fails with

> ERROR: ArgumentError: Unrecognized argument type: fmpz.

Instead, with `coeffs = [1, 2]` this works.

Reply:
What happens if you do `BigInt(x)`?

Reply:
`coeffs = [BigInt(1), BigInt(2)]` seems to work as input for 

> ptd = Polymake.fulton.TDivisor(COEFFICIENTS=coeffs)

Reply:
Not sure if it is realistic that an invariant divisor is bigger than `2^63`, but with `BigInt` we would be on the "safe side". Not sure if it has performance implications though.

Reply:
If we stay with Int, we need an overflow check. But I really think we should store fmpz, and ToricDivior should accept these; and then either Polymake needs to learn to accept these here (cc @benlorenz ) or we convert to BigInt. (If the coeffs are not accessed externally later, we can of course immediately work with BigInt here, whatever

Reply:
I think I would prefer fmpz. (Similar issue: rays of a fan currently return `Polymake.rational`, for which I would also prefer fmpz).

Reply:
The coefficients can be accessed externally (for all it matters).

Reply:
Agreed

Reply:
`Polymake.fulton.TDivisor` is a direct interface to a pure `polymake` construction (from autogenerated code) which cannot deal with fmpz types. Until there is some kind of Oscar wrapper around the TDivisor / or until we can inject some kind of automatic conversion there, this needs to be passed polymake compatible types as input, i.e. Int64, BigInt, Polymake.Rational, Polymake.Integer.
Inside polymake this will be stored as a polymake vector of GMP rationals anyway, so just cast them to some big enough type:
```julia
julia> ptd = Polymake.fulton.TDivisor(COEFFICIENTS=Polymake.Vector{Polymake.Integer}(xcoeffs))
type: TDivisor

COEFFICIENTS
        1 2

julia> ptd.COEFFICIENTS
pm::Vector<pm::Rational>
1 2
```

Reply:
Thanks for the explanation, Benjamin